app/views/layouts/_left_nav.html.haml

```diff
@@ -0,0 +1,12 @@
+%div{ class: 'small-2 columns' }
+  .row
+    %div{ class: 'small-12 medium-10 large-8 columns left_nav' }
+      %ul{ class: 'vertical menu'}
```

Comment:
Hash attribute should end with one space before the closing brace<br>Avoid defining `class` in attributes hash for static class names